packages/flutter/lib/src/widgets/image.dart

```diff
@@ -436,6 +436,8 @@ class Image extends StatefulWidget {
   /// will be rendered to the constraints of the layout or [width] and [height]
   /// regardless of these parameters. These parameters are primarily intended
   /// to reduce the memory usage of [ImageCache].
+  ///
+  /// This keeps a cache of the file which should be manually evicted.
```

Comment:
It's not clear to me what this additional doc is supposed to tell me. What is the context? (That context should also be added to the doc comment).